ocs_ci/helpers/managed_services.py

```diff
@@ -28,21 +28,17 @@ def verify_provider_topology():
     6. Verify OSD count
     7. Verify OSD CPU and memory
 
+    Args:
+        size(str): cluster size
+
     """
     # importing here to avoid circular import
     from ocs_ci.ocs.resources.storage_cluster import StorageCluster, get_osd_count
 
-    size = f"{config.ENV_DATA.get('size', 4)}Ti"
+    size = size if size is not None else f"{config.ENV_DATA.get('ms_size')}"
```

Comment:
I recommend to keep using size config option and just to update it at the end of the upscale test so it contains true value. The ms_size option is not always used but size is default place to look for cluster size.
Also I recommend to change the name from ms_size to upscale_size or something similar. ms_size doesn't reflect the purpose of the config variable.

Reply:
done